includes/Checker/Abstract_Check_Runner.php

```diff
@@ -489,6 +487,18 @@ final public function get_checks_to_run() {
 		return $collection->to_map();
 	}
 
+	/**
+	 * Initializes the runtime environment so that runtime checks can be run against a separate set of database tables.
+	 *
+	 * @since n.e.x.t
```

Comment:
Note that we don't use this pattern in this plugin, there's no script to replace them. Can just manually set it to 1.3.0.

```suggestion
	 * @since 1.3.0
```